src/employee-invoice/dtos/employee-salary.dto.ts

```diff
@@ -1,13 +1,13 @@
-import { IsDateString, IsEmail, IsNotEmpty, IsNumber, IsString } from "class-validator";
-import { Base } from "src/interface/base";
+import { IsDateString, IsEmail, IsNotEmpty, IsNumber, IsOptional, IsString } from "class-validator";
+import { Base } from "../../interface/base";
+
+export class EmployeeSalaryDto extends Base {
```

Comment:
FE has only this 5 things

Emp id, Overtime hour, Absent Hour, advanceWithdrawal and Incentive